pxr/imaging/hdSt/materialXFilter.cpp

```diff
@@ -1055,37 +1052,30 @@ _ReplaceFilenameInput(
     hdNetwork->nodes[hdTerminalNodePath].
         inputConnections[terminalTextureConn.upstreamOutputName] =
             {terminalTextureConn};
-
 }
 
-// Gather the Material Params from the glslfx ShaderPtr
```

Comment:
Not technically a GLSLFX shader yet - it's just generated by MaterialX and then later wrapped in GLSLFX.